### ISSUE TITLE
cloud search service 7.10.2 support in versions < otc terraform provider v1.31.2

### ISSUE THREAD

**Issue by rhtot** [opentelekomcloud/terraform-provider-opentelekomcloud#2019]
Hi there,
we had some issues regarding our CSS cluster, and we ended up upgrading to v7.10.2 based on a recommendation by OTC. Currently we are running an old version of the provider, which supports the following version when defined in the datastore block:

```
  datastore {
    type    = "elasticsearch"
    version = "7.10.2"
  }
```
Now we are planning a provider upgrade, which would mean >v1.31.2, however newer versions throw an error (as expected based on the documentation segment quoted below):

```
│ Error: expected datastore.0.version to be one of [7.6.2 7.9.3], got 7.10.2
│ 
│   with module....elasticsearch.opentelekomcloud_css_cluster_v1.elasticsearch_cluster,
│   on ../../elasticsearch.tf line 25, in resource "opentelekomcloud_css_cluster_v1" "elasticsearch_cluster":
│   25:     version = "7.10.2"
```
The documentation side:

opentelekomcloud_css_cluster_v1

[version] - Engine version. The value can be 7.6.2 or 7.9.3. The default value is 7.6.2.

The question from my side is the following:

Will v7.10.2 become supported by the newer provider versions soon, or we should consider rolling back to v7.9.3 with the resource?

**Comment by vladimirvshivkov** [comment]
Hey @rhtot , 
I made a quick fix that will be available on 21 December.

**Comment by rhtot** [comment]
Hi @vladimirvshivkov 
Sounds good, we'll test when it's ready. Thanks a lot for the quick response.

**Comment by vladimirvshivkov** [comment]
@rhtot 
Fix released in the new version v1.32.1, check, please.

**Comment by rhtot** [comment]
@vladimirvshivkov 
Works like a charm, thank you!
